tests/components/lcn/conftest.py

```diff
@@ -104,9 +103,23 @@ def create_config_entry_myhome():
 
 async def init_integration(hass, entry):
     """Set up the LCN integration in Home Assistant."""
+    lcn_connection = MockPchkConnectionManager(
+        entry.data[CONF_IP_ADDRESS],
+        entry.data[CONF_PORT],
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+        settings={
+            "SK_NUM_TRIES": entry.data[CONF_SK_NUM_TRIES],
+            "DIM_MODE": pypck.lcn_defs.OutputPortDimMode[entry.data[CONF_DIM_MODE]],
+        },
+        connection_id=entry.entry_id,
+    )
+
     entry.add_to_hass(hass)
-    await hass.config_entries.async_setup(entry.entry_id)
-    await hass.async_block_till_done()
+    with patch("pypck.connection.PchkConnectionManager", return_value=lcn_connection):
+        await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+    return lcn_connection
```

Comment:
```suggestion
    return manager.return_value
```

Reply:
See comment above. This would return a `MagicMock` but we need an instance of `MockPchkConnectionManager`.
Therefore, I would suggest to keep it as it is.